src/commands/mirror_node.mjs

```diff
@@ -45,6 +49,8 @@ export class MirrorNodeCommand extends BaseCommand {
     }
 
     valuesArg += ` --set hedera-mirror-node.enabled=true --set hedera-explorer.enabled=${deployHederaExplorer}`
+    valuesArg += ` --set hedera-mirror-node.importer.config.hedera.mirror.importer.parser.record.sidecar.enabled=true`;
```

Comment:
If this is a must, then we should update it in the helm chart: https://github.com/hashgraph/full-stack-testing/blob/main/charts/fullstack-deployment/values.yaml#L289C15-L289C22